src/PointwiseFunctions/AnalyticData/CurvedWaveEquation/RegularSphericalWaveKerrSchild.hpp

```diff
@@ -74,10 +102,11 @@ class RegularSphericalWaveKerrSchild : public MarkAsAnalyticData {
   using options =
       tmpl::list<BlackHoleMass, BlackHoleSpin, BlackHoleCenter, WaveProfile>;
   static constexpr OptionString help{
-      "A regular spherical wave solution of the covariant wave equation"
-      " with a Kerr black hole (in Kerr-Schild coordinates) as background.\n"
-      "WARNING: This solution should only be used to set initial data."};
-  static std::string name() noexcept { return "RegularSphWaveKerr"; };
+      "A regular spherical wave solution of the covariant wave equation\n"
```

Comment:
"A regular spherical wave with a Kerr..."  (i.e. omit solution of ... equation) .   And you can omit the WARNING as it is marked as analytic data

Reply:
changed